pkgs/by-name/mi/microsocks/package.nix

```
@@ -0,0 +1,29 @@
+{
+  stdenv,
+  fetchFromGitHub,
+  lib
+}:
+stdenv.mkDerivation rec {
```

```suggestion
{
  stdenv,
  fetchFromGitHub,
  lib
}:

stdenv.mkDerivation rec {
```

It's usual to skip a line after parameters.